src/Numerics/CUDA/cuda_inverse.h

```diff
@@ -29,6 +29,26 @@ cublas_inverse (cublasHandle_t handle,
                 int N, int rowStride, int numMats,
                 bool useHigherPrecision = true);
 
+void
+cublas_lemma_mats (cublasHandle_t handle,
```

Comment:
cuda_inverse.h/cu is for matrix inversion.
cublas_lemma_mats, cublas_ainv_row, cublas_smw_update are not generic wrapper functions of cublas.
Please move them to /sandbox/opt/cleanup/atillack/src/QMCWaveFunctions/Fermion

Reply:
No production code should be in sandbox. Put distinct functionality in different files or simply rename the cuda_inverse.h file e.g. cuda_matrices.h

Reply:
The path I put previously was wrong. It was just my local path. Functions under Numerics should not be only used a specific algorithm. So cuda_matrices.h is not good. Please create a new set of .h and .cu file under src/QMCWaveFunctions/Fermion.

Reply:
OK

Reply:
@ye-luo @prckent Done.